runtime/src/iree/tooling/vm_util.cc

```diff
@@ -245,44 +244,42 @@ Status ParseToVariantList(iree_hal_allocator_t* device_allocator,
   return OkStatus();
 }
 
+#define IREE_PRINTVARIANTLIST_CASE_I(S, B, V)                                 \
+  case IREE_VM_VALUE_TYPE_I##S:                                               \
+    iree_string_builder_append_format(B, "i" #S "=%" PRIi##S "\n", (V).i##S); \
```

Comment:
all the append methods may fail and will need to be wrapped in IREE_RETURN_IF_ERROR

Reply:
done